core-galleon-pack/src/main/resources/feature_groups/host-interfaces.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="host-interfaces" xmlns="urn:jboss:galleon:feature-group:1.0">
+    <feature spec="host.interface">
+        <param name="interface" value="public"/>
+        <param name="inet-address" value="${jboss.bind.address:127.0.0.1}"/>
+    </feature>
```

Comment:
I see some differences interface element ordering between the galleon output and legacy. Perhaps this is why.

The traditional order is management, public, private, unsecure.  Not all four are used in all configs of course; I don't think private is used at all in core.

This isn't a huge problem but OTOH if it's as simple as moving this element ^^^ below the 'management' one then that will eliminate a diff to legacy.

Reply:
I tried it and confirmed that moving this done eliminates the diff.

Reply:
However, changing core-galleon-pack/src/main/resources/feature_groups/interfaces.xml to put public after management does not result in public coming after management in standalone.xml.

So, I'd rather have consistent interface order between the galleon-produced configs than consistency with the legacy configs in some. If we can get both, great, if not the way you have it now where public comes first is fine.

Reply:
For standalone it's because the deps kick in and make the public added first. But if you re-order the groups in standalone group by moving the interfaces before socket-binding-group, will preserve order of the interfaces. I'm gonna correct this to be in sync with the legacy. Thanks.